share/ruby-build/jruby-10.0.0.1

```diff
@@ -0,0 +1,2 @@
+require_java 8
```

Comment:
Shouldn't this require a newer Java version? https://github.com/rbenv/ruby-build/commit/4cdc5a720f5410b3128bf64458d42c65ad5568c0

Reply:
I have fixed that and re-pushed, but where does this come from? I just used the update script. Does it look at the current Java in PATH and somehow use that?